packages/overdrive/lib/components/Toaster/Toast.tsx

```diff
@@ -88,11 +90,23 @@ const InternalToastProvider = ({ children }) => {
 		<ToastControllerContext.Provider value={addToast}>
 			{children}
 			<Portal>
-				<div className={styles.root}>
-					{toasts.map((item) => (
-						<Toast key={item.id} {...item} remove={removeToast} />
-					))}
-				</div>
+				<Box
+					width="full"
```

Comment:
This can't be full width, it breaks the clicks of children.  Also further down, `justifyContents` has no effect here as its parent isnt a gird/flex.

![image](https://user-images.githubusercontent.com/599459/88248545-e3855f80-cce4-11ea-838b-3f92a75d8712.png)